samples/Makefile-src

```diff
@@ -12,29 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-HTM_TARGETS = $(patsubst %.htm, ../../viewers/%-viewer.htm, $(wildcard *.htm))
-HTML_TARGETS = $(patsubst %.html, ../../viewers/%-viewer.html, $(wildcard *.html))
-XHTML_TARGETS = $(patsubst %.xhtml, ../../viewers/%-viewer.xhtml, $(wildcard *.xhtml))
-
 # eval followed by space is a silly trick to match a space
-PACKAGES = $(subst $(eval) ,|,$(wildcard ../../packages/*.zip))
```

Comment:
I guess the `eval` comment is no longer relevant.

Reply:
Yep - fixed in 80795da